coremltools/converters/mil/mil/input_type.py

```diff
@@ -285,7 +285,7 @@ def type_domain(self):
 
     @type_domain.setter
     def type_domain(self, val):
-        msg = "type_domain must be a tuple of builtin types"
+        msg = f"type_domain {val} must be a tuple of builtin types"
```

Comment:
This is an unrelated change but made errors during the complex lowering passes a little easier to debug.